use_cases/eluc/prescriptors/prescriptor.py

```diff
@@ -23,4 +25,17 @@ def predict_metrics(self, context_actions_df: pd.DataFrame) -> tuple:
         Returns a dataframe of ELUC and change.
         """
         raise NotImplementedError
+    
+    def _compute_percent_changed(self, context_actions_df: pd.DataFrame) -> pd.DataFrame:
+        """
+        Calculates percent of land changed by prescriptor.
+        """
```

Comment:
Moved this here because it doesn't require any extra knowledge or implementation.

Reply:
According to PEP 8:
> _single_leading_underscore: weak "internal use" indicator

`compute_percent_changed` is used by other classes, especially sub classes, so I'd remove the `_`